Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -917,7 +917,7 @@ WebPageProxy::WebPageProxy(PageClient& pageClient, WebProcessProxy& process, Ref
         process.setIgnoreInvalidMessageForTesting();
 #endif
 
-#if ENABLE(MEDIA_SESSION_COORDINATOR) && HAVE(GROUP_ACTIVITIES)
+#if USE(APPLE_INTERNAL_SDK) && ENABLE(MEDIA_SESSION_COORDINATOR) && HAVE(GROUP_ACTIVITIES)
```

Comment:
Can you help me understand why this is needed? GroupActivities is a public framework, and I'd expect the SDK check to be written inside Platform.h instead (here, probably in the definition of `ENABLE_MEDIA_SESSION_COORDINATOR`).